app/services/sla_breach_service.rb

```diff
@@ -11,11 +11,27 @@ def breach_threshold_date
   # attention_needed_since is present and is before breach threshold
   # returns data in the format { vita_partner_id: breach_count }
   def attention_needed_breaches
-    Client.sla_tracked.group(:vita_partner_id).where(Client.arel_table[:attention_needed_since].lteq(breach_threshold_date)).count(:id)
+    Client.sla_tracked
+      .where(Client.arel_table[:attention_needed_since].lteq(breach_threshold_date))
+      .group(:vita_partner_id)
+      .count(:id)
   end
 
   # clients who messaged us and have not been responded to _with a message, call or email_ within the breach threshold
   def outgoing_communication_breaches
-    Client.sla_tracked.group(:vita_partner_id).where(Client.arel_table[:first_unanswered_incoming_interaction_at].lteq(breach_threshold_date)).count(:id)
+    Client.sla_tracked
+      .where(Client.arel_table[:first_unanswered_incoming_interaction_at].lteq(breach_threshold_date))
+      .group(:vita_partner_id)
+      .count(:id)
+  end
```

Comment:
It looks like this moves the position of the goup call in this query. Totally non-blocking, but out of curiosity, does this change the result of the query or is the change just for readability/clarity?

Reply:
It's for readability -- @sniemeyer13 had provided some feedback in a previous PR that the query was a little hard for her to understand, so breaking it into two lines and putting the "formatting" part of the query at the end helped, I thought. (When I accidentally moved group after count, it makes the query fail).

Also, the reformatting was also partially a result of product asking "will we be able to see the actual clients that are in breach -- with this refactor, it preps us really well to somehow extract the group/count part of the query into something reuseable when we need it, but potentially reuse the guts of the query to display clients who are in breach.